common/src/test/scala/uk/ac/wellcome/models/ConceptTest.scala

```diff
@@ -0,0 +1,69 @@
+package uk.ac.wellcome.models
+
+import org.scalatest.{FunSpec, Matchers}
+import org.scalatest.prop.TableDrivenPropertyChecks._
+import uk.ac.wellcome.test.utils.JsonTestUtil
+import uk.ac.wellcome.utils.JsonUtil._
+
+class ConceptTest extends FunSpec with Matchers with JsonTestUtil {
+
+  val testCases = Table(
+    ("concept", "expectedJson"),
+    (
+      Concept(label = "Woodwork"),
+      s"""{
+        "type": "Concept",
+        "ontologyType": "Concept",
+        "label": "Woodwork",
+        "qualifierType": null,
+        "qualifiers": []
+      }"""
+    ),
+    (
+      QualifiedConcept(
+        label = "Dangerous diseases",
+        concept = Concept(
+          label = "disease",
+          qualifiers = List(
+            Concept(
+              qualifierType = "general-subdivision",
+              label = "dispersion & direction"
+            )
+          )
+        )
+      ),
+      s"""{
+        "type": "QualifiedConcept",
+        "ontologyType": "QualifiedConcept",
+        "label": "Dangerous diseases",
+        "concept": {
+          "label": "disease",
+          "ontologyType": "Concept",
+          "qualifierType": null,
+          "qualifiers": [
+            {
+              "label": "dispersion & direction",
+              "ontologyType": "Concept",
+              "qualifierType": "general-subdivision",
+              "qualifiers": []
+            }
+          ]
+        }
+      }"""
+    )
+  )
+
+  it("serialises Concepts to JSON") {
+    forAll(testCases) { (concept: AbstractConcept, expectedJson: String) =>
```

Comment:
why is this a property test? If the two rows in the table test different things, they should be separate tests with different names to clarify what it is that we are testing

Reply:
I usually do parametrised tests like this for reduced boilerplate.